phpBB/adm/style/acp_search_index.html

```diff
@@ -30,7 +30,7 @@ <h1>{{ lang('ACP_SEARCH_INDEX') }}</h1>
 
 		{{ backend.S_HIDDEN_FIELDS }}
 
-		<legend>{{ lang('INDEX_STATS') }}{{ lang('COLON') }} {{ backend.NAME }} {% if backend.S_ACTIVE %}({{ lang('ACTIVE') }}) {% endif %}</legend>
+		<legend>{{ lang('INDEX_STATS') ~ lang('COLON') }} {{ backend.NAME }} {% if backend.S_ACTIVE %}({{ lang('ACTIVE') }}) {% endif %}</legend>
 
 		<table class="table1">
 			<caption>{{ backend.NAME }} {% if backend.S_ACTIVE %}({{ lang('ACTIVE') }}) {% endif %}</caption>
```

Comment:
Not sure why those parenthesis are hardcoded?
`({{ lang('ACTIVE') }})`

The leading space here seems like its not necessary?
` {% endif %}</caption>`